sign/clin_sig/meta.yaml

```diff
@@ -0,0 +1,15 @@
+name: clin_sig
+type: "column consistent of strings, separated by ',' "
+description: |
+  This spell visualizes the clinical significance, given in clinvar significance terms
+link-to-url: https://www.ncbi.nlm.nih.gov/clinvar/
```

Comment:
Is this used in the docs? Maybe rather annotate the mention of clinvar in the description above with a markdown link.